docs-src/docs/kognic_apis.md

```diff
@@ -38,3 +44,11 @@ The credentials file, including the Kognic client ID and the Kognic client secre
 
 ![Generate API credentials app screenshot](../static/img/generate-api-credentials.png)
 
+
+## Proxy Configuration
+
+If your organizations' network policy requires HTTP(S) traffic to be proxied out via a specific host, then you should configure this via your OS or execution environment. `kognic-io` uses Python's `urllib` which [will pick up the proxy configuration from your OS and environment variables](https://docs.python.org/3/library/urllib.request.html#urllib.request.getproxies).
+
```

Comment:
I would add for example, and probably note that this is an internal server, and to get connection info reach out to your internal support org.